src/sentry/tagstore/snuba/backend.py

```diff
@@ -495,7 +509,7 @@ def get_tag_value_paginator(self, project_id, environment_id, key, query=None,
             orderby=order_by,
             # TODO: This means they can't actually paginate all TagValues.
             limit=1000,
-            referrer='tagstore.get_tag_value_paginator',
+            referrer=referrer if referrer is not None else 'tagstore.get_tag_value_paginator_for_projects',
```

Comment:
I don't mind if we just hardcode referrer to `'tagstore.get_tag_value_paginator_for_projects'` here. There are lots of other cases in tagstore where the referrer is set to the name of some helper method rather than the original tagstore method that was called and thats fine.